src/rx5808-pro-diversity/state.cpp

```diff
@@ -43,7 +43,7 @@ namespace StateMachine {
         currentHandler = newHandler;
 
         if (newHandler)
-            lastHandler->onEnter();
+			newHandler->onEnter();
```

Comment:
I've fixed this in my most recent push which I think is the only merge conflict. Well spotted! Took me ages to find and fix this myself, haha.